lib/cog/chat/hipchat/template_processor.ex

```diff
@@ -24,6 +24,8 @@ defmodule Cog.Chat.HipChat.TemplateProcessor do
     do: "<code>#{text}</code>"
   defp process_directive(%{"name" => "fixed_width_block", "text" => text}),
     do: "<pre>#{text}</pre>"
+  defp process_directive(%{"name" => "link", "text" => text, "url" => url}),
```

Comment:
Well I should say that the markdown style links did. There is another head that deals with links that have no text. I should probably group these together :|